src/index.coffee

```diff
@@ -34,8 +34,8 @@ class DatadogPlugin
   # The lower the return value, the more requests failed (HTTP 5xx)
   # Treat redirects as OK
   getOkPercentage: (metrics) ->
-    percentage = (metrics['response.2xx'] + metrics['response.3xx']) * 100 \
-     / metrics['scenarios.completed']
+    percentage = (metrics['response.2xx'][0] + metrics['response.3xx'][0]) * 100 \
```

Comment:
this was making percentage > 100 as artillery is counting 200 and 300 as separate requests even when with in the same scenario.